pkg/config/storage.go

```diff
@@ -0,0 +1,74 @@
+package config
+
+// StorageConfig provides configs for various storage backends
+type StorageConfig struct {
+	CDN   *CDNConfig   `validate:""`
+	Disk  *DiskConfig  `validate:""`
+	GCP   *GCPConfig   `validate:""`
+	Minio *MinioConfig `validate:""`
+	Mongo *MongoConfig `validate:""`
+	RDBMS *RDBMSConfig `validate:""`
+}
+
+func setDefaultTimeouts(s *StorageConfig, defaultTimeout int) {
```

Comment:
do we need this function if we already have `default` struct tags in the Config?

Reply:
this sets timeouts for storage backends to the global timeout value if they haven't been specified. Since it's not a static value (it's whatever the global timeout is set to), I couldn't think of a better way to do this!